selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java

```diff
@@ -27,6 +27,11 @@
 import org.openqa.selenium.By;
 
 import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.SelenideElement;
+import static  com.codeborne.selenide.Selectors.*;
+import org.openqa.selenium.By;
```

Comment:
This import already exist.
And others added imports seems useless.

Reply:
code formatted